server/server.js

```diff
@@ -11,3 +11,24 @@
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+
+var http = require('http');
+var eDate = require('./enhancedDate');   //requires enhancedDate.js module we created
+var trucks = require('./trucks'); //requires trucks.js
+
+function handleRequest(request, response) {
+    eDate.setDate();
+    var dayName = eDate.getDayName();
+    var monthName = eDate.getMonthName();
+    var date = eDate.getDate(true);
+    var message = "Today is " + dayName + ", " + monthName + " " + date + ". Here are the available food trucks:";
+    //var truckList = "foo"
+    var truckList = trucks.filterByDay(dayName);
+    response.writeHead(200, { 'Content-Type': 'text/plain'});
+    response.write(message + " " + truckList);
```

Comment:
truckList in it's array form will just write out something like [object object], [object object]. Instead, you want to build up a string first where you've looped through truckList, extracting the relevant information from each object in the array. For example:

`var string = '';
var i;
var l;
for (i = 0, l = trucksLIst.length; i < l; i += 1) {
  string += trucksList[i] + '\n';
}`